shared/haml/hoc_onetrust_cookie_scripts.haml

```diff
@@ -0,0 +1,12 @@
+-# OneTrust Cookies Consent Notice scripts for hourofcode.com
+- cookie_script_env = DCDO.get('onetrust_cookie_scripts', 'test')
```

Comment:
I was having 2nd thoughts about defaulting to `test`. Drone doesn't run UI tests or anything, so do we don't need this feature turned on for Drone builds, right? We mostly need it turned on when `eyes` tests are running against the test servers. And if they are running against the test servers, we can control the DCDO configuration by editing the DynamoDB table which the test servers rely on. Thoughts?

Reply:
Yeah that does make sense. The issues that came up in the past were caught by eyes tests, not by Drone, so I agree Drone probably doesn't need this turned on at all.